tests/test_box_Box.py

```diff
@@ -112,6 +112,38 @@ def test_wrap_multiple_images(self):
         points = np.array(points)
         npt.assert_allclose(box.wrap(points)[0, 0], -2, rtol=1e-6)
 
+    def test_out_default(self):
+        box = freud.box.Box(2, 2, 2, 1, 0, 0)
+        points = [[10, -5, -5], [0, 0.5, 0]]
+        npt.assert_allclose(box.wrap(points)[0, 0], -2, rtol=1e-6)
```

Comment:
Could we check the entire array is correct, instead of just one value? Applies to all tests below.